packages/main/src/plugin/default-configuration.spec.ts

```diff
@@ -0,0 +1,148 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import * as path from 'node:path';
+
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+
+import { DefaultConfiguration } from './default-configuration.js';
+
+// mock constants
+vi.mock('/@api/configuration/constants.js', () => ({
+  SYSTEM_DEFAULTS_FILE_MAC: '/Library/Application Support/com.podman.desktop/default-settings.json',
+  SYSTEM_DEFAULTS_FILE_WINDOWS_DIR: 'PodmanDesktop',
+  SYSTEM_DEFAULTS_FILE_WINDOWS_FILE: 'default-settings.json',
+  SYSTEM_DEFAULTS_FILE_LINUX: '/usr/share/podman-desktop/default-settings.json',
+}));
+
+// mock the fs module as we don't want to check the read file / exists / promises, but rather
+vi.mock('node:fs', () => ({
+  readFileSync: vi.fn(),
+  existsSync: vi.fn(),
+  promises: {
+    readFile: vi.fn(),
+  },
+}));
```

Comment:
```suggestion
vi.mock(import('node:fs'));
```

You can simplify the mock here